test/ctl/ctl_unittest.cpp

```diff
@@ -21,35 +21,35 @@ TEST_F(test, ctl_debug_read_from_string) {
 
     int value = 0;
     ctl_query(ctl_handler, NULL, CTL_QUERY_PROGRAMMATIC,
-              "debug.heap.alloc_pattern", CTL_QUERY_READ, &value);
+              "debug.heap.alloc_pattern", CTL_QUERY_READ, &value, 0);
```

Comment:
my comment from the prev. PR is not resolved:

> whenever size is used, please add a test with various values (user may give some input we may not expect, e.g. on wrong assumptions on how this func works) or mark a TODO for adding tests in the future

e.g. here you can simply add a repeated queries with non-zero values

Reply:
Fixed.